src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -124,653 +126,602 @@ def _configure_db_create_params(
+                   help='List of disabled alerts.',
+                   nargs='+')
+
+        c.argument('email_addresses',
+                   arg_group=notification_arg_group,
+                   options_list=('--email-addresses',),
+                   help='List of email addresses that alerts are sent to.',
+                   nargs='+')
+
+        c.argument('email_account_admins',
+                   arg_group=notification_arg_group,
+                   options_list=('--email-account-admins',),
+                   help='Whether the alert is sent to the account administrators.',
+                   arg_type=get_enum_type(SecurityAlertPolicyEmailAccountAdmins))
+
+        # TODO: use server default
+
+    #####
+    #           sql db transparent-data-encryption
+    #####
+    with self.argument_context('sql db tde') as c:
+        c.argument('database_name',
+                   options_list=('--database', '-d'),
+                   required=True,
+                   help='Name of the Azure SQL Database.')
+
+    with self.argument_context('sql db tde set') as c:
+        c.argument('status',
+                   options_list=('--status'),
+                   required=True,
+                   help='Status of the transparent data encryption.',
+                   arg_type=get_enum_type(TransparentDataEncryptionStatus))
+
+    ###############################################
+    #                sql dw                       #
+    ###############################################
+    with self.argument_context('sql dw') as c:
+        c.argument('database_name', options_list=('--name', '-n'),
+                   help='Name of the data warehouse.')
+
+        c.argument('server_name', arg_type=server_param_type)
+
+        c.argument('max_size_bytes', options_list=('--max-size',),
+                   type=SizeWithUnitConverter('B', result_type=int),
+                   help='The max storage size of the data warehouse. If no unit is specified, defaults'
+                   'to bytes (B).')
+
+        c.argument('requested_service_objective_name',
+                   options_list=('--service-objective',),
+                   help='The service objective of the data warehouse.')
+
+        c.argument('collation',
+                   options_list=('--collation',),
+                   help='The collation of the data warehouse.')
+
+    with self.argument_context('sql dw create') as c:
+        _configure_db_create_params(c, Engine.dw, CreateMode.default)
+
+    with self.argument_context('sql dw show') as c:
+        # Service tier advisors and transparent data encryption are not included in the first batch
+        # of GA commands.
+        c.ignore('expand')
+
+    # Data Warehouse restore will not be included in the first batch of GA commands
+    # (list_restore_points also applies to db, but it's not very useful. It's
+    # mainly useful for dw.)
+    # with ParametersContext(command='sql dw restore-point') as c:
```

Comment:
Was this previously commented out? I would convert it so that is in the right format if it is later included.

Reply:
Yep, was previously commented out. Okay, let me convert it as well, though the verification will have to be done later

Reply:
You can just delete these commented out commands.

Reply:
done